docs/en/query_language/select.md

```diff
@@ -450,23 +450,25 @@ See the standard [SQL JOIN](https://en.wikipedia.org/wiki/Join_(SQL)) descriptio
 
 #### Multiple JOIN
 
-Performing queries, ClickHouse rewrites multiple joins into the combination of two-table joins and processes them sequentially. If there are four tables for join ClickHouse joins the first and the second, then joins the result with the third table, and at the last step, it joins the fourth one.
+When running queries, ClickHouse rewrites multiple joins as a combination of two-table joins and processes them sequentially. If there are four tables to join, ClickHouse combines the first and second table, then joins the result with the third, then finally with the fourth.
 
-If a query contains `WHERE` clause, ClickHouse tries to push down filters from this clause into the intermediate join. If it cannot apply the filter to each intermediate join, ClickHouse applies the filters after all joins are completed.
+If a query contains the `WHERE` clause, ClickHouse tries to push filters from this clause down to the intermediate join. If it cannot apply the filter to each intermediate join, ClickHouse applies the filters after all joins are completed.
```

Comment:
* "pushdown" is a common term for this optimization and is hard to catch if it's split so far in a sentence
* it's pushed not "into" join, but  "through" join to subquery before it